compiler/analyzer.ml

```diff
@@ -638,6 +743,42 @@ and check_if env i t e =
   } in
   env', Sast.Expr(Sast.If(ifdecl), const)
 
+(* Distrubutions *)
+(* TO DO: UPDATE COLLECT_CONSTRAINTS TO ACCOUNT FOR DIST *)
```

Comment:
@akalicki @alexandramedway - it actually doesn't need to be updated b/c the dist type is so simple - i'll push a change w/ this comment deleted